WebApiCoreSeed.Domain/Services/Interfaces/IUserService.cs

```diff
@@ -1,19 +1,25 @@
-﻿namespace WebApiCoreSeed.Domain.Services.Interfaces
-{
-    using System;
-    using System.Threading.Tasks;
-    using WebApiCoreSeed.Data.Models;
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
 
-    /// <summary>
-    /// Service responsible of handling users
-    /// </summary>
-    public interface IUserService
-    {
-        /// <summary>
-        /// Gets a user by Id
-        /// </summary>
-        /// <param name="userId">Id of the user to be retrieved</param>
-        /// <returns>A <see cref="User"/> object if the user is found, otherwise null</returns>
-        Task<User> GetByIdAsync(Guid userId);
-    }
+namespace WebApiCoreSeed.Domain.Services.Interfaces
+{
+    /// <summary>
+    /// Service responsible of handling users
+    /// </summary>
+    public interface IUserService
+    {
+        /// <summary>
+        /// Gets a user by Id
+        /// </summary>
+        /// <param name="userId">Id of the user to be retrieved</param>
+        /// <returns>A <see cref="User"/> object if the user is found, otherwise null</returns>
+        Task<User> GetByIdAsync(Guid userId);
+
+        Task<int> CreateAsync(User user);

```

Comment:
Super-minor: do you think you could add XML docks for these methods? Since the other method has it, maybe we can keep it consistent.

Reply:
Will do!